crmsh/utils.py

```diff
@@ -2198,4 +2198,59 @@ def check_ssh_passwd_need(hosts):
         if rc != 0:
             return True
     return False
+
+
+def check_port_open(host, port):
+    import socket
+
+    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
+        if sock.connect_ex((host, port)) == 0:
+            return True
+        else:
+            return False
+
+
+def valid_port(port):
+    return int(port) >= 1024 and int(port) <= 65535
+
+
+def use_qdevice():
```

Comment:
I don't like much this name `use_qdevice`. This function is to check if qdevice is configured, right?
I would rename to `is_qdevice_configured` or similar as it returns a bool actually.
But maybe I'm missing something

Reply:
changed!